swagger.yaml

```diff
@@ -5176,6 +5176,49 @@ paths:
           schema:
             $ref: '#/definitions/error'
 
+  /v2/clusters/{cluster_id}/actions/allow-add-nodes:
```

Comment:
@mhrivnak @eranco74 @filanov Others...
How do you feel about this name?  I thought "transform" was too vague.  Maybe "allow-add-workers" is better?  Other options?

Reply:
@nmagnezi FYI

Reply:
It's a bit confusing to see that we have both node, host and agent not sure what is right but we should consider a single standard

Reply:
So maybe allow-add-workers is better in that sense of avoiding node/host/agent.  Is allow-adding-workers even better?

Reply:
allow-add-workers sounds the best, changing